modules/nw-configure-sysctl-interface-sriov-network.adoc

```diff
@@ -0,0 +1,171 @@
+
+.Procedure
+
+. Create the `SriovNetwork` custom resource (CR) for the additional SR-IOV network attachment and insert the `metaPlugins` configuration, as in the following example CR. Save the YAML as the file `sriov-network-interface-sysctl.yaml`.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: onevalidflag <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  resourceName: policyoneflag <3>
+  networkNamespace: sysctl-tuning-test <4>
+  ipam: '{ "type": "static" }' <5>
+  capabilities: '{ "mac": true, "ips": true }' <6>
+  metaPlugins : | <7>
+    {
+      "type": "tuning",
+      "capabilities":{
+        "mac":true
+      },
+      "sysctl":{ 
+         "net.ipv4.conf.IFNAME.accept_redirects": "1"
+      }
+    }
+----
+<1> A name for the object. The SR-IOV Network Operator creates a NetworkAttachmentDefinition object with same name.
+<2> The namespace where the SR-IOV Network Operator is installed.
+<3> The value for the spec.resourceName parameter from the SriovNetworkNodePolicy object that defines the SR-IOV hardware for this additional network.
+<4> The target namespace for the SriovNetwork object. Only pods in the target namespace can attach to the additional network.
+<5> A configuration object for the IPAM CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<6> Optional: The capabilities to configure for this additional network. You can specify "{ "ips": true }" to enable IP address support or "{ "mac": true }" to enable MAC address support.
+<7> Optional: The metaPlugins parameter is used to add additional capabilities to the device. In this use case set the `type` field to `tuning`. Specify the interface-level network `sysctl` you want to set in the `sysctl` field. 
+
+. Create the `SriovNetwork` resource:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-interface-sysctl.yaml
+----
+
+.Verifying that the `NetworkAttachmentDefinition` CR is successfully created
+
+* Confirm that the SR-IOV Network Operator created the `NetworkAttachmentDefinition` CR by running the following command.
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions -n <namespace> <1>
+----
+<1> Replace <namespace> with the networkNamespace you specified in the SriovNetwork object. For example `sysctl-tuning-test`. 
++
+.Example output
+[source,terminal]
+----
+NAME                                  AGE
+onevalidflag                          14m
+----
++
+[NOTE]
+====
+There might be a delay before the SR-IOV Network Operator creates the CR.
+====
+
+.Verifying that the additional SR-IOV network attachment is successful
+
+To verify that the tuning CNI is correctly configured and the additional SR-IOV network attachment is attached, do the following:
+
+. Create a pod such as `examplepod.yaml` similar to the following:.
++
+[source,terminal]
+----
```

Comment:
Same here, Pod yaml format looks different from other resources. It's like grayed out. Not sure if it should be like that.